storybook/storybook-react/src/components/Icon/Icon.stories.tsx

```diff
@@ -4,21 +4,23 @@
  */
 
 import { Icon } from '@amsterdam/design-system-react/src'
-import { Email } from '@amsterdam/design-system-react-icons'
-import { ComponentMeta, ComponentStory } from '@storybook/react'
+import { Email } from '@amsterdam/design-system-react-icons/src'
+import { Meta, StoryFn } from '@storybook/react'
 
 import '@amsterdam/design-system-css/src/icon/icon.scss'
 
 export default {
   title: 'Icon',
   id: 'icon',
   component: Icon,
-} as ComponentMeta<typeof Icon>
+} as Meta<typeof Icon>
```

Comment:
Do we run into issues using the `satisfies` keyword here?

Reply:
No, the migrate script just doesn't use that by default.